<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.291" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2025-06-10T16:48:17" tests="0" file="cypress\e2e\automation_OrangeHRM.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Fluxo de cadastro no OrangeHRM" timestamp="2025-06-10T16:48:17" tests="1" time="0.288" failures="1">
    <testcase name="Fluxo de cadastro no OrangeHRM Deve realizar o cadastro de um novo colaborador com sucesso" time="0.000" classname="Deve realizar o cadastro de um novo colaborador com sucesso">
      <failure message="cy.contains(...).should(...).or is not a function" type="TypeError"><![CDATA[TypeError: cy.contains(...).should(...).or is not a function
    at Context.eval (webpack://automation_orangehrm/./cypress/e2e/automation_OrangeHRM.cy.js:33:7)
    at runnable.fn (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:141791:19)
    at callFn (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:160869:21)
    at Runnable.run (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:160856:7)
    at <unknown> (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:166928:30)
    at PassThroughHandlerContext.finallyHandler (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:4042:23)
    at PassThroughHandlerContext.tryCatcher (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1777:23)
    at Promise._settlePromiseFromHandler (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1489:31)
    at Promise._settlePromise (https://opensource-demo.orangehrmlive.com/__cypress/runner/cypress_runner.js:1546:18)]]></failure>
    </testcase>
  </testsuite>
</testsuites>